webpack/components/Search/Search.js

```diff
@@ -12,6 +12,12 @@ import {
   AUTOSEARCH_WHILE_TYPING,
 } from '../../scenes/Settings/SettingsConstants';
 
+// Don't trigger auto-search until you've typed an operator and begun to type a value
+const searchContainsOperator = (search = '') => {
+  const operators = ['>=', '<=', '>', '<', '~', '^', '!^', '=', '!=', '~', '!~'];
```

Comment:
Does it make sense to add a check that search doesn't end with 'and' or 'or'?

Reply:
I tried this, but by the time 'and' or 'or' is added to a search it already contains an operator, so it doesn't help anything..

Reply:
We'd probably want to split on 'and' and 'or' and apply the same operator check on all different parts of search or maybe just the last part after the and/or.